### PR TITLE
Create works about components

### PR DESCRIPTION
**Pull request by Artufelii**
Add new works and about components to the index.html file and the corresponding styles.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -77,3 +89,180 @@ body {
...
+  color: #6070ff;
+}
+
+.technologies li {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+}
+
+.see {
```
Comment (huseyinbiyik) [inline css/styles.css:185]: Kindly add hover and pressed styles to your "See Project" buttons. In this way you can show your pixel perfect implementation skill and also this states will increase the user experience of the web page.
Screenshot 2023-03-24 at 11 02 32 PM

File: css/styles.css [hunk 2]
```diff
@@ -77,3 +89,180 @@ body {
...
+}
+
+.info {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.info-title {
```
Comment (huseyinbiyik) [inline css/styles.css:138]: Great job with font-weight, family and size 👍  But let's make sure we have the same color with the Figma value.
Screenshot 2023-03-24 at 11 07 54 PM
Screenshot 2023-03-24 at 11 10 25 PM

File: css/styles.css [hunk 3]
```diff
@@ -77,3 +89,180 @@ body {
...
+  gap: 25px;
+}
+
+.extract {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.extract p {
```
Comment (huseyinbiyik) [inline css/styles.css:164]: Kindly make sure you are using the same color for your paragraph. Check the difference below.
Screenshot 2023-03-24 at 11 10 42 PM
Screenshot 2023-03-24 at 11 10 54 PM

File: index.html [hunk 4]
```diff
@@ -52,6 +52,164 @@ <h1>I'm Arturo Alvarez.Glad to see you!</h1>
...
+							<li>JavaScript</li>
+						</ul>
+						<a href="https://farmaciajesus.onrender.com" class="see">See project</a>
+					</div>
+				</div>
+			</div>
+		</section>
+		<section class="about">
+			<div class="title">
+				<p>About Myself</p>
```
Comment (huseyinbiyik) [inline index.html:139]: Kindly use semantic hx element for this title. It has an importance in the page and we can show it with hiearchical hx tags. Kindly give a hierarchically proper hx value for this title. It is important in terms of SEO and accessibility.
- Also please fix the font color.
Screenshot 2023-03-24 at 11 32 41 PM

File: index.html [hunk 5]
```diff
@@ -52,6 +52,164 @@ <h1>I'm Arturo Alvarez.Glad to see you!</h1>
...
+			</div>
+			<div class="me position">
+					<p>
+							Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+					</p>
+			</div>
+			<div class="contact position">
+				<p>LET'S CONNECT</p>
```
Comment (huseyinbiyik) [inline index.html:147]: Kindly use a hx heading tag for this heading element. As I mentioned before, it has a huge importance in terms of SEO and accessibility.

File: index.html [hunk 6]
```diff
@@ -52,6 +52,164 @@ <h1>I'm Arturo Alvarez.Glad to see you!</h1>
...
+							<img src="/assets/Linkedin icon.svg" alt="Linkedin logo">
+						</a>
+					</li>
+					<li>
+						<a href="https://github.com/Artufelii" class="a">
+							<img src="/assets/Vector.svg" alt="Github logo">
+						</a>
+					</li>
+				</ul>
+				<a href="/assets/ArturoAlvarez-Resume.pdf" download="ArturoAlvarez-Resume" class="see">Get my resume</a>
```
Comment (huseyinbiyik) [inline index.html:165]: Kindly add hover and pressed styles to your "Get my resume" link. In this way you can show your pixel perfect implementation skill and also this states will increase the user experience of the web page.

Screenshot 2023-03-24 at 11 23 06 PM

File: index.html [hunk 7]
```diff
@@ -52,6 +52,164 @@ <h1>I'm Arturo Alvarez.Glad to see you!</h1>
...
+			</div>
+			<div class="skills">
+				<div class="skill">
+					<div class="name">
+						<p>Languages</p>
```
Comment (huseyinbiyik) [inline index.html:170]: The technology categories ("Languages", "Frameworks" and "Skills") needs to be in a ul, each of them in a li. Kindly fix them to make your web page more semantic.

File: index.html [hunk 8]
```diff
@@ -52,6 +52,164 @@ <h1>I'm Arturo Alvarez.Glad to see you!</h1>
...
+					</li>
+					<li>
+						<a href="https://www.linkedin.com/in/arturo-alvarez-martinez-ab9b77151/" class="a">
+							<img src="/assets/Linkedin icon.svg" alt="Linkedin logo">
+						</a>
+					</li>
+					<li>
+						<a href="https://github.com/Artufelii" class="a">
+							<img src="/assets/Vector.svg" alt="Github logo">
+						</a>
```
Comment (huseyinbiyik) [inline index.html:162]: Good job with adding some of the social media links. But let's add all of them we see on the Figma template. As I mentioned before we need to stick to the design and code it pixel perfectly.
Screenshot 2023-03-24 at 11 29 00 PM
Screenshot 2023-03-24 at 11 31 08 PM

File: index.html [hunk 9]
```diff
@@ -52,6 +52,164 @@ <h1>I'm Arturo Alvarez.Glad to see you!</h1>
...
+			</div>
+		</section>
+		<section class="about">
+			<div class="title">
+				<p>About Myself</p>
+			</div>
+			<div class="me position">
+					<p>
+							Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+					</p>
```
Comment (huseyinbiyik) [inline index.html:144]: Good job with using p tag here 👍  But let's fix the font color. You can see the color value below 👍 
Screenshot 2023-03-24 at 11 33 33 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @Artufelii 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Eternalgratis — APPROVED** [review]
# Status: Approved 🎈🎈🎈
Hi @Artufelii , 

Well done on implementing the changes required by the previous reviewer. In my opinion, your project is ready for submission!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 
Developer-512px-10


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**